### ISSUE TITLE
rjulia.so not found

### ISSUE THREAD

**Issue by st-lenk86** [armgong/RJulia#21]
Hi,

I am trying to install RJulia under Ubuntu 15.04 (64 bit). My installed Julia version is 0.3.11 and my R version is 3.2.2. Everything goes just fine until R tests if the new package can be loaded, Then R is apparently not able to find rjulia.so . The error message looks like this:

```
** R
** demo
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
Error : .onLoad failed in loadNamespace() for 'rjulia', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: cannot load shared object '~/R/x86_64-pc-linux-gnu-library/3.2/rjulia/libs/rjulia.so':
  libjulia.so: Cannot load shared object file: File or directory not found
Error: Loading failed
Execution halted
ERROR: loading failed
* removing ‘~/R/x86_64-pc-linux-gnu-library/3.2/rjulia’
E: Command failed (1)
```

What is going wrong? Help is very much appreciated.

**Comment by armgong** [comment]
i don't have  Ubuntu 15.04 right now ,but the error message show  libjulia.so not loaded. 
could you run this command in tty and give me the output 
julia -E 'abspath(Sys.dlpath("libjulia"))'

**Comment by st-lenk86** [comment]
The command returned: 

```
/usr/lib/x86_64-linux-gnu/julia/libjulia.so
```

**Comment by armgong** [comment]
try these lines in R console ,and see what happens

.juliaLib <- function()
    gsub("\"", "",
         system('julia -E "abspath(Sys.dlpath(\\\"libjulia\\\"))"', intern = TRUE))

.ldjulia <<- .juliaLib()
.ldjulia
 dyn.load(.ldjulia, local = FALSE)

**Comment by st-lenk86** [comment]
I entered these lines into the R console and tried to install RJulia again, but it fails with the same error as before.

**Comment by armgong** [comment]
I install ubuntu 15.04 in docker ,and R is 3.1.2 julia is 0.3.10 , install ok,but when julia_init() there are problem, because jl_init() in julia api have some bug.
as a temp fix you can 
 ln -s /usr/lib/x86_64-linux-gnu/julia/libjulia.so /usr/lib/libjulia.so
 ln -s /usr/lib/x86_64-linux-gnu/julia /usr/lib/julia

**Comment by armgong** [comment]
after try julia 0.4 rc and 0.5 nightly and rjulia on ubuntu 15.04,  I will not fix this bug ( pls use the temp fix) , because julia 0.4 will release soon ,after that rjulia will not support 0.3 anymore.

**Comment by st-lenk86** [comment]
Thank you for your help. Now it is working.
